tests/e2e/default/sms/messages-breadcrumbs.wdio-spec.js

```diff
@@ -0,0 +1,98 @@
+const utils = require('../../../utils');
+const commonElements = require('../../../page-objects/default/common/common.wdio.page');
+const loginPage = require('../../../page-objects/default/login/login.wdio.page');
+const userFactory = require('../../../factories/cht/users/users');
+const placeFactory = require('../../../factories/cht/contacts/place');
+const personFactory = require('../../../factories/cht/contacts/person');
+const messagesPo = require('../../../page-objects/default/sms/messages.wdio.page');
```

Comment:
Could you please rename this constant to `messagesPage` or `messagesPageObject`?

Reply:
Sure, done !